Gemfile

```diff
@@ -25,7 +25,6 @@ gem 'active_model_serializers'
 gem 'omniauth-google-oauth2'
 gem 'will_paginate'
 gem 'acts-as-taggable-on', '~> 6.0'
-gem 'omniauth-census', git: "https://github.com/turingschool-projects/omniauth-census"
```

Comment:
I was wondering what this was before, I thought it was something that had maybe been in the project before we started. Did you just add this and realize we didnt need it?

Reply:
I'm not sure what this was but I didn't use it so I deleted it but the tests didn't fail. 
I added this `gem 'omniauth-github', github: 'omniauth/omniauth-github', branch: 'master'`